client/components/AuthorCard.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { StarRating } from "./StarRating";
+import Link from "next/link";
+import Image from "next/image";
+import { CiEdit } from "react-icons/ci";
+import dayjs from "dayjs";
+
+export const AuthorCard = ({ author }) => {
```

Comment:
Let's work on segregation of `components` with `entities`. This will give us better folder structure and easy understanding.